src/components/BulkLookup.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import { onMount, createEventDispatcher } from "svelte";
+  import { state as authState } from "../auth.js";
+  import { bulkId } from "../couch.js";
+
+  const dispatch = createEventDispatcher();
+  let token = $authState.token;
+  export let metaId = "";
+  let metaIdList = metaId.split("\n");
+  let contentarr = [];
+  let content;
+  let metaList = [];
+  let contentcheck;
+  export let id, db;
+  export let label = "Please provide a label for this Component";
+
+  async function bulkIdLookup(event) {
+    try {
+      metaList = await bulkId(token, db, metaIdList);
```

Comment:
Since you've bound `metaIdList` to the textarea, you're not actually splitting anything when this gets sent to the `bulkId` call.